### PR TITLE
/api/userinfo display all user imformation

### PR DESCRIPTION
**Pull request by Kanav-7**
https://github.com/felicity-iiith/contest-portal-backend/issues/22

### CODE REVIEW

File: src/routes/main.js [hunk 1]
```diff
@@ -10,12 +10,21 @@ async function ctrl(ctx) {
...
 const routes = [
   {
     method: 'get',
     path: '/',
     handler: ctrl
   },
+  {
+    method: 'get',
+    path: '/userinfo',
+    handler: [isAuthenticated, isUnlocked,userinfo]
```
Comment (meghprkh) [inline src/routes/main.js]: No need for isUnlocked here. isUnlocked is for checking questions. Please test this via postman once

**Review by anubhabsen — DISMISSED** [review]
Fix indents, import isAuthenticated

File: src/routes/main.js (hunk 1 above)
Comment (Kanav-7) [reply src/routes/main.js]: @meghprkh  added that by mistake in the second commit. Check now.

File: src/routes/main.js [hunk 2]
```diff
@@ -10,12 +10,21 @@ async function ctrl(ctx) {
...
 const routes = [
   {
     method: 'get',
     path: '/',
     handler: ctrl
   },
+  {
+    method: 'get',
+    path: '/userinfo',
+    handler: userinfo
```
Comment (meghprkh) [inline src/routes/main.js]: Need isAuthenticated still

**Pull request merged**